x-pack/plugins/security_solution/public/app/deep_links/index.ts

```diff
@@ -265,13 +265,6 @@ export const securitySolutionsDeepLinks: SecuritySolutionDeepLink[] = [
             }),
             path: `${HOSTS_PATH}/events`,
           },
-          {
-            id: SecurityPageName.hostsExternalAlerts,
-            title: i18n.translate('xpack.securitySolution.search.hosts.externalAlerts', {
-              defaultMessage: 'External Alerts',
-            }),
-            path: `${HOSTS_PATH}/externalAlerts`,
```

Comment:
@paulewing There doesn't appear to be any "url state" for the `Show only external alerts` checkbox. I wouldn't hold up this PR for it, but if this state existed:

- It would be possible to share links to the `Hosts` | `Network` | `Users` > `Events` tab, filtered by external alerts
- It would be possible to redirect all existing links to the (now deleted) `External alerts` tabs to the `Events` tab